src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2010.ProvideCodeSnippets_Valid.sarif

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner",
+          "version": "1.0"
+        }
+      },
+      "versionControlProvenance": [
+        {
+          "repositoryUri": "https://github.com/microsoft/sarif-sdk"
+        }
+      ],
+      "results": [
+        {
+          "ruleId": "TST0001",
+          "level": "error",
+          "message": {
+            "text": "Some testing occurred."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "src/test.c"
+                },
+                "region": {
+                  "startLine": 3,
+                  "snippet": {"text": "region code snippet"}
+                },
+                "contextRegion": {
+                  "startLine": 2,
+                  "endLine": 4,
+                  "snippet": { "text": "context region code snippet. More context here." }
+                }
+              }
+            }
```

Comment:
>} [](start = 12, length = 1)

Add a valid test case where it's just a region with a snippet, no contextRegion.

Reply:
I take it back. That will cause a `result` to appear in your 'Valid' output file.

---
In reply to: [448028963](https://github.com/microsoft/sarif-sdk/pull/1953#discussion_r448028963) [](ancestors = 448028963)

Reply:
larry i added it and trhen decided to remove it - reason: then we get violation for rule#2011. it was basically  a copy of 2011_invalid. i can add it back - it just felt unnecessary. I can add it back.

---
In reply to: [448028963](https://github.com/microsoft/sarif-sdk/pull/1953#discussion_r448028963) [](ancestors = 448028963)

Reply:
aah thank u :) u read my mind!

---
In reply to: [448029654](https://github.com/microsoft/sarif-sdk/pull/1953#discussion_r448029654) [](ancestors = 448029654,448028963)